### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by asoback**
Made some resources a little more accessible by including links to them in the README. Information was pulled from the website, documentation, and the contributing pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,20 @@
...
-Source code repository for the QIIME 2 framework. Visit [https://qiime2.org](https://qiime2.org) to learn more about the QIIME 2 project.
+Source code repository for the QIIME 2 framework. 
+
+QIIME 2™ is a powerful, extensible, and decentralized microbiome bioinformatics platform that is free, open source, and community developed. With a focus on data and analysis transparency, QIIME 2 enables researchers to start an analysis with raw DNA sequence data and finish with publication-quality figures and statistical results.
+
+Visit [https://qiime2.org](https://qiime2.org) to learn more about the QIIME 2 project.
+
+## Installation
+
+Detailed [instructions](https://docs.qiime2.org/2019.4/install/) are available in the documentation.
```
Comment (thermokarst) [inline README.md]: How about we drop the `/2019.4....` part, we will never remember to update that with each release.

File: README.md [hunk 2]
```diff
@@ -2,4 +2,20 @@
...
+
+## Users
+Head to the [user docs](https://docs.qiime2.org/2019.4/) for help getting started, core concepts, tutorials, and other resources.
```
Comment (thermokarst) [inline README.md]: How about we drop the `/2019.4....` part, we will never remember to update that with each release.

File: README.md [hunk 3]
```diff
@@ -2,4 +2,20 @@
...
+
+Just have a question? Please ask it in our [forum](https://forum.qiime2.org/c/user-support).
+
+## Developers
+Please visit the [contributing page](https://github.com/asoback/qiime2/blob/master/.github/CONTRIBUTING.md) for more information on contributions, documentation links, and more.
```
Comment (thermokarst) [inline README.md]: Oops, wrong repo here, should be https://github.com/qiime2/qiime2

File: README.md [hunk 4]
```diff
@@ -10,12 +10,12 @@ Visit [https://qiime2.org](https://qiime2.org) to learn more about the QIIME 2 p
 
 ## Installation
 
-Detailed [instructions](https://docs.qiime2.org/2019.4/install/) are available in the documentation.
+Detailed [instructions](https://docs.qiime2.org/install/) are available in the documentation.
```
Comment (thermokarst) [inline README.md]: Oh sorry, that link still won't work --- needs to be https://docs.qiime2.org

**Comment by thermokarst** [comment]
Looks good to me - thanks! We are in a code freeze right now, will merge once we thaw. Thanks!

**Comment by asoback** [comment]
Thanks! I just squashed my commits as well, so now there is only one.

**Pull request merged**